ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Client.hs

```diff
@@ -35,27 +35,25 @@ import Control.Monad.Class.MonadTimer
 
 import Network.TypedProtocol.Core
 
-import Ouroboros.Network.Block (Point)
 import Ouroboros.Network.Protocol.ChainSync.Type
 
 
 -- | A chain sync protocol client, on top of some effect 'm'.
 -- The first choice of request is within that 'm'.
-newtype ChainSyncClient header tip m a = ChainSyncClient {
-    runChainSyncClient :: m (ClientStIdle header tip m a)
+newtype ChainSyncClient header point tip m a = ChainSyncClient {
+    runChainSyncClient :: m (ClientStIdle header point tip m a)
   }
 
 -- | A chain sync client which never sends any message.
 --
-chainSyncClientNull :: MonadTimer m => ChainSyncClient header tip m a
+chainSyncClientNull :: MonadTimer m => ChainSyncClient header point tip m a
 chainSyncClientNull = ChainSyncClient $ forever $ threadDelay 43200 {- one day in seconds -}
 
-{-# DEPRECATED chainSyncClientNull "Use Ouroboros.Network.NodeToClient.chainSyncPeerNull" #-}
```

Comment:
I saw you removed this warning for `localTxSubmissionClientNull` too. So they're both no longer deprecated?

Reply:
Oh well spotted. I should not have done that in this PR. That should be a separate patch.